Docs/Configuration.tex

```diff
@@ -3092,16 +3092,30 @@ \subsection{Boot Properties}\label{miscbootprops}
       \item \texttt{<ENTRY\_PATH>.icns} --- icon near the entry file with appended \texttt{.icns} extension.
     \end{itemize} \medskip
 
-    For all other entries OpenCore will try to load a volume icon and fallback to the default icon:
+    For all other entries OpenCore will try to load a volume icon by searching
+    as follows, and will fallback to the default icon otherwise:
     \begin{itemize}
     \tightlist
-      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} volume directory for APFS (if present).
-      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} root for APFS (otherwise).
+      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} per-volume directory
+      (\texttt{/System/Volumes/Preboot/-GUID-/} within macOS) for APFS (if present).
```

Comment:
Hmm, I slightly do not like the `/System/Volumes/Preboot/-GUID-/` part. `Preboot` is the role of a volume inside an APFS container. How It matters little on how it is mounted in macOS, and in fact you can mount it anywhere, not just here. Maybe better to word it like this?

```
      \item \texttt{.VolumeIcon.icns} file at \texttt{Preboot} volume in per-volume directory
       (\texttt{/System/Volumes/Preboot/\{GUID\}/} when mounted at default location within
       macOS) for APFS (if present).
```

Reply:
Applied